bins/recipe-runner/src/runner.c

```diff
@@ -641,7 +653,7 @@ GglError runner(const RecipeRunnerArgs *args) {
     }
     default:
         GGL_LOGE("Failed to get noProxyAddresses from config.");
-        return ret;
+        break;
```

Comment:
Similar comment as above. One additional question - what happens if the proxy URL is set but the noProxyAddress value is not set?